[BUG]: Exit the app drawer search screen and go to home screen makes the app crash

### Describe the bug

I tried debugging it myself but it's a very strange bug

```
error: Uncaught exception

android.view.InflateException: Binary XML file line #45: Binary XML file line #45: Error inflating class TextView
Caused by: android.view.InflateException: Binary XML file line #45: Error inflating class TextView
Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 5: TypedValue{t=0x2/d=0x7f0403ce a=5 r=0x7f060449}
	at android.content.res.TypedArray.getColorStateList(TypedArray.java:546)
	at android.widget.TextView.readTextAppearance(TextView.java:3708)
	at android.widget.TextView.<init>(TextView.java:1015)
	at android.widget.TextView.<init>(TextView.java:925)
	at android.widget.TextView.<init>(TextView.java:921)
	at app.lawnchair.LawnchairLayoutFactory$constructorMap$2.invoke(LawnchairLayoutFactory.kt:18)
	at app.lawnchair.LawnchairLayoutFactory$constructorMap$2.invoke(LawnchairLayoutFactory.kt:18)
	at app.lawnchair.LawnchairLayoutFactory.onCreateView(LawnchairLayoutFactory.kt:29)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:772)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:863)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
	at com.android.launcher3.allapps.WorkProfileManager.attachWorkModeSwitch(WorkProfileManager.java:174)
	at com.android.launcher3.allapps.WorkProfileManager.updateCurrentState(WorkProfileManager.java:158)
	at com.android.launcher3.allapps.WorkProfileManager.reset(WorkProfileManager.java:141)
	at com.android.launcher3.allapps.ActivityAllAppsContainerView.reset(ActivityAllAppsContainerView.java:502)
	at com.android.launcher3.allapps.ActivityAllAppsContainerView.reset(ActivityAllAppsContainerView.java:468)
	at com.android.launcher3.Launcher.onStateSetEnd(Launcher.java:1280)
	at com.android.launcher3.uioverrides.QuickstepLauncher.onStateSetEnd(QuickstepLauncher.java:484)
	at com.android.launcher3.uioverrides.QuickstepLauncher.onStateSetEnd(QuickstepLauncher.java:189)
	at com.android.launcher3.statemanager.StateManager.onStateTransitionEnd(StateManager.java:400)
	at com.android.launcher3.statemanager.StateManager.-$$Nest$monStateTransitionEnd(Unknown Source:0)
	at com.android.launcher3.statemanager.StateManager$1.onAnimationSuccess(StateManager.java:377)
	at com.android.launcher3.anim.AnimationSuccessListener.onAnimationEnd(AnimationSuccessListener.java:40)
	at android.animation.Animator$AnimatorListener.onAnimationEnd(Animator.java:552)
	at android.animation.AnimatorSet.endAnimation(AnimatorSet.java:1294)
	at android.animation.AnimatorSet.doAnimationFrame(AnimatorSet.java:1079)
	at android.animation.AnimationHandler.doAnimationFrame(AnimationHandler.java:146)
	at android.animation.AnimationHandler.access$100(AnimationHandler.java:37)
	at android.animation.AnimationHandler$1.doFrame(AnimationHandler.java:54)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1032)
	at android.view.Choreographer.doCallbacks(Choreographer.java:845)
	at android.view.Choreographer.doFrame(Choreographer.java:777)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1020)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:205)
	at android.app.ActivityThread.main(ActivityThread.java:6993)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:884)

```

It appears to originate here when trying to inflate one layout

https://github.com/LawnchairLauncher/lawnchair/blob/14-dev/src%2Fcom%2Fandroid%2Flauncher3%2Fallapps%2FWorkProfileManager.java#L174-L175

On line 45 of 'work_mode_fab': this
https://github.com/LawnchairLauncher/lawnchair/blob/14-dev/res%2Flayout%2Fwork_mode_fab.xml#L45

 In other words, along the lines that define this style 

https://github.com/LawnchairLauncher/lawnchair/blob/14-dev/res%2Fvalues%2Fstyles.xml#L381

But looking at the documentation, this was supposed to be compatible since API 14

https://developer.android.com/reference/android/R.style#TextAppearance_DeviceDefault_DialogWindowTitle



### Steps to reproduce

Steps to reproduce the behavior:
1. Go to 'Seach bar in App Drawer'
2. Wait the content to be loaded (apps etc..)
3. Click outside to minimize the keyboard without leaving search screen
4. Click on 'back' button to go to Home Screen
5. See error


### Expected behavior

Not crash

### Screenshots

https://github.com/LawnchairLauncher/lawnchair/assets/93414086/af8c8906-9e33-4cc8-91a4-b6cddb72dee0

This also happens when I close the search by dragging it down

https://github.com/LawnchairLauncher/lawnchair/assets/93414086/5fd6d3ce-61d1-4003-a14b-87c0859ab09d


### Device information

- Device: old LG
- OS: Android 9
- App version: 14-dev latest debug buil


### Additional context

_No response_

--- a/src/com/android/launcher3/allapps/WorkProfileManager.java
+++ b/src/com/android/launcher3/allapps/WorkProfileManager.java
@@ -166,7 +166,8 @@ public class WorkProfileManager implements PersonalWorkSlidingTabStrip.OnActiveP
      */
     public boolean attachWorkModeSwitch() {
         if (!mAllApps.getAppsStore().hasModelFlag(
-                FLAG_HAS_SHORTCUT_PERMISSION | FLAG_QUIET_MODE_CHANGE_PERMISSION)) {
+                FLAG_HAS_SHORTCUT_PERMISSION | FLAG_QUIET_MODE_CHANGE_PERMISSION)
+            || !Utilities.ATLEAST_Q) {
             Log.e(TAG, "unable to attach work mode switch; Missing required permissions");
             return false;
         }

